frontend/public/config.yml

```
@@ -383,6 +383,40 @@ collections:
                   },
                 ],
             }
+      - label: 'Contact settings and socials'
+        name: 'contact_settings'
+        file: 'frontend/src/content/pages/contact-settings.md'
+        fields:
+          - {
+              label: 'email',
```

yamllint (1.35.1)

[error] wrong indentation: expected 12 but found 14

(indentation)